edx_proctoring/api.py

```diff
@@ -1870,13 +1884,19 @@ def _get_practice_exam_view(exam, context, exam_id, user_id, course_id):
     """
     Returns a rendered view for the practice Exams
     """
+    user = USER_MODEL.objects.get(id=user_id)
+
     student_view_template = None
 
     attempt = get_exam_attempt(exam_id, user_id)
 
     attempt_status = attempt['status'] if attempt else None
 
-    if not attempt_status:
+    user_has_active_account = user.is_active
```

Comment:
#NIT why create this variable when you can just do `not user.is_active` below? This line seems redundant.